test/e2e/Apps/BasicJava/extensions.csproj

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <!-- This reference will be dynamically updated by build-e2e-tests.ps1 as part of test setup. -->
     <!-- Do not commit changes to this line unless necessary -->
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="3.4.1" />
```

Comment:
Please do not commit these changes, as it causes issues with the pipelines - can you revert all the extensions.csproj files from the e2e suite?